examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -33,6 +38,9 @@
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                     proj=True, baseline=(None, 0), preload=True,
                     reject=dict(grad=4000e-13, eog=150e-6))
+# Pull 2**n points to speed up computation
+epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
+                     epochs.events, tmin, epochs.event_id)
```

Comment:
I think the relevant lines are in epochs.py: in the __init__ at 290, get_data at 1154 and the cropping function 1435 

The issue is the first index. E.g.:

```python
tmin = -1.
tmax = tmin + 1023. / raw.info['sfreq']
epochs = mne.Epochs(raw, events, event_id, tmin, tmax)
print(epochs.times)
epochs.crop(tmin, tmax)
print(epochs.times)
```
gives:
[-1.00064103 -0.99897607 -0.99731111 ...,  0.69928325  0.70094821
  0.70261317]
[-0.99897607 -0.99731111 -0.99564615 ...,  0.69928325  0.70094821
  0.70261317]

I feel like the outputs should be the same, but in the second case the first index gets lopped off.